### DISCUSSION TITLE
Compatibility with Pipez

### DISCUSSION THREAD

**Discussion by gBonvenuto** [fnuecke/oc2#219] [Ideas]
I come from ComputerCraft, and there we can move items from one inventory to another. In OpenComputers (as far as my research goes) it seems computers can only look at the contents of chests while robots can move items.

But when you need to move an item to a far distance, making the robots walk such distances and sometimes complex paths, that becomes inviable, my workaround has been using redstone to deal with the items pipes from the [Pipez mod](https://www.curseforge.com/minecraft/mc-mods/pipez). I wonder if there could be a way for us to access the Pipez menu through the computer. For example, toggle blacklist/whitelist, toggle recieve/send items, and **specially change the filter**.

Since the mod is still in alpha, I understand that this suggestion is not urgent. Anyways, thank you for this amazing mod

**Comment by CoolCat467** [comment]
I think the proper way to go about having this happen is having the Pipez mod developers add OpenComputers interface support, but other people might know more about this.
